### DISCUSSION TITLE
Is "/api/auth/[...nextauth]" omitted from the middleware?

### DISCUSSION THREAD

**Discussion by AntonioErdeljac** [nextauthjs/next-auth#9398] [Help]
Hi everyone, I've been using next-auth in production for a while now, and Im noticing very rare errors (5 errors on 50k success logs) in my production log regarding `/api/auth` route.

In my exploration to resolve them, I started wondering whether the problem could be in my middleware which is explicitly protecting all `/api/:path*` routes.

`
export const config = {
  matcher: ["/api/:path*"],
};
`

But when logging `req.nextUrl.pathname` I am never receiving any logs for `/api/auth/[...nextauth]` routes, for example `/api/auth/providers` never gets logged as a hit in my middleware. 

I've gone and explicitly allowed all `/api/auth...` routes just in case, but I want to confirm my doubts.

**Are `/api/auth/[...nextauth]` routes omitted from the middleware?**

Much love to all creators and matinainers of this amazing project ❤️

**Comment by bilalmohib** [comment]
Thats true I am also not able to log the routes
